data/gegno/gegno intro missions.txt

```diff
@@ -0,0 +1,331 @@
+# Copyright (c) 2022 by Saugia
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Gegno Vi: First Contact"
+	landing
+	source "Var Oti"
+	on offer
+		conversation		
+			branch "second encounter"
+				has "Giaru Gegno: Quarg Contact: offered"
+			branch "return encounter"
+				has "Gegno Vi: First Contact: deferred"
+				
+			`As you descend towards the surface of this seemingly uninhabited world, your radar systems start to pick up several faint signatures below in every direction. None of the signatures change their behavior as you continue, but you decide to keep your ship a few thousand meters above the surface while staying wary in alien territory. You take the time to carefully look around, but there doesn't seem to be anything of note besides the cracked rock formations and valleys of sand.`
+			`Suddenly, your radar systems pick up a larger signature further up ahead on the horizon, and in the distance is an odd bulky rock-like structure. Once you get close enough, the structure begins to resemble ships you've seen flying around nearby systems, although it is half sunken into the sand.`
+			choice
+				`	(Land near the ship and investigate.)`
+					goto land
+				`	(Continue to search the surface.)`
+					goto search
+			label land
+			`	Still keeping your guard up, you carefully land your ship nearby. Most of the faint radar signatures from earlier have disappeared out of range, so it seems there's nothing coming after you at the moment. Exiting your ship in an environment suit with your sidearm on hand, you walk towards the alien ship while remaining alert.`
+			`	On closer inspection, the alien ship appears a bit larger than you thought it was - around the size of a Firebird. The dark-brown hull looks to be made from materials that you aren't familiar with, and it has several metallic-looking parts both hatched in crevices and protruding outward. Oddly enough, it seems perfectly intact, but the way it is angled into the sand suggests that the ship has been here for some time. There doesn't seem to be any activity around it, and you aren't picking up any life signs nearby. Leaving the site, you decide to head back to your ship and take another look around the area.`
```

Comment:
Gegno ships, especially the Vi, are slightly larger all-around than human counterparts since the Gegno themselves are larger beings. Kind of like the Sheragi, but not to that extent. I figured being up close to one would show this scaling a bit better than from a distance, but if it's negligible I can remove that description.